apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Tests/Logging/GoogleLoggerTest.cs

```diff
@@ -146,5 +148,36 @@ public void Log()
             logger.Log(LogLevel.Error, 0, _logMessage, s_exception, Formatter);
             mockConsumer.VerifyAll();
         }
+
+        [Fact]
+        public void Log_Trace()
+        {
+            string traceId = "105445aa7843bc8bf206b12000100f00";
+            string fullTraceName = TraceTarget.ForProject(_projectId).GetFullTraceName(traceId);
+
+            Predicate<IEnumerable<LogEntry>> matcher = (l) =>
```

Comment:
Nit: I'd rename l to logEnties and remove the parentheses around the lambda parameter:

```csharp
Predicate<IEnumerable<LogEntry>> matcher = logEntries =>
...
```

Not a blocker though.